client/login/wp-login/index.jsx

```diff
@@ -576,13 +571,17 @@ export class Login extends Component {
 			);
 		}
 
+		const isLostPassword =
+			currentRoute === '/log-in/lostpassword' || currentRoute === '/log-in/jetpack/lostpassword';
```

Comment:
I'm also working on this in parallel for https://github.com/Automattic/wp-calypso/pull/104367, but no worries, this PR should be merged first.

I think we can send the `action` prop instead to `HeadingSubText`, which will have the value `lostpassword` for the lost-password screen. Can you try it with that instead? :-)

Reply:
Thanks @chriskmnds for checking again!

> I think we can send the action prop instead to HeadingSubText, which will have the value lostpassword for the lost-password screen. Can you try it with that instead? :-)

Nice, that would be way better than this check (I added it last minute, I noticed password reset pages had the ToS).

I just tried the `action` prop, but it was `null` in every case. Before diving in deeper, and delay this PR any further I think I'll go for a merge now. Happy to address this in a subsequent PRs.

Reply:
Interesting. It worked for me, but I wasn't testing the Woo flow. Glad that we merged this 🤔 

The `action` comes from some login context and was used in other places including for Woo login e.g. in footer. I'll dig around.

Thanks for seeing this PR to completion! 🎉